platform/windows/key_mapping_windows.cpp

```diff
@@ -335,6 +336,8 @@ static _WinTranslatePair _scancode_to_keycode[] = {
 	{ KEY_UNKNOWN, 0 }
 };
 
+static std::unordered_set<unsigned int> _extended_keys({ VK_INSERT, VK_DELETE, VK_HOME, VK_END, VK_PRIOR, VK_NEXT, VK_LEFT, VK_UP, VK_RIGHT, VK_DOWN });
```

Comment:
We have our own container types in Godot and don't use STL ones, see `core/list.h`, `core/set.h` and `core/vector.h`.

While it doesn't make a big difference here, I'd ask to change it to use a Godot container (not sure which is best for the use case but `Set<int>` or `Vector<int>` should work fine for such a simple set).